build.groovy

```diff
@@ -495,7 +495,45 @@ def doDocs(Rule rule)
         throw new TablesawException("Unable to run sphinx-build")
 }
 
+//---------------------------------------------------------------------------
+//Build container
+dockerBuild = new SimpleRule("docker-build").setDescription("Build a Docker image")
+.addDepend(tarRule)
+.setMakeAction("doDockerBuild")
 
+def doDockerBuild(Rule rule)
+{
+  def tag = getDockerTag()
+  command = "docker build -t ${tag} --build-arg VERSION=${version}-${release} ."
+  saw.exec(command)
+}
+
+def getDockerTag()
+{
+  env = System.getenv()
```

Comment:
Change this to:
`def registry = saw.getProperty("DOCKER_REGISTRY", "");

	if ( registry != "" ) {
		registry += "/"
	}`

Reply:
I give up on formatting that, but you get the idea.  saw.getProperty pulls properties from various sources so you can specify the value from a -D or the environment or from a tablesaw.properties file

Reply:
Perfect. Will do, thanks.